parm/archive/gdas_restarta.yaml.j2

```
@@ -15,6 +15,19 @@ gdas_restarta:
         {% for iaufhr in IAUFHRS if iaufhr != 6 %}
         - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}atmi{{ "%03d" % iaufhr }}.nc"
         {% endfor %}
+        
+        # Soil DA increments
+        {% if DO_GSISOILDA %}
+        {% if DOIAU %}
+        {% for itile in range(6) %}
+        - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/sfc_inc.tile{{ itile+1 }}.nc"
+        {% endfor %}
+        {% else %}
+        {% for itile in range(6) %}
+        - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/sfci006.tile{{ itile+1 }}.nc"
```

I'd prefer these to match other `itile` loops:

```suggestion
        {% for itile in range(1,7) %}
        - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/sfc_inc.tile{{ itile }}.nc"
        {% endfor %}
        {% else %}
        {% for itile in range(1,7) %}
        - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/sfci006.tile{{ itile }}.nc"